wadas/ui/mainwindow.py

```diff
@@ -448,6 +452,19 @@ def update_info_widget(self):
                 str(OperationMode.cur_operation_mode.last_classified_animals_str)
             )
 
+        if (db := DataBase.get_instance()) and db.enabled:
```

Comment:
Since getting a db and check if it's enable is a recurring pattern, maybe you can create a get_enabled_db method which gives you back a database instance if it exists AND it's enabled. ;)